content/learning-paths/servers-and-cloud-computing/cca-veraison/attestation-verification.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Use the verification service
+weight: 5
+
+### FIXED, DO NOT MODIFY
+layout: learningpathall
+---
+
+## Attestation Verification Service for Pre-Silicon CCA Platforms
+Linaro provides an attestation verifier service for pre-silicon CCA platforms, such as the Fixed Virtual Platform (FVP). This service is available publicly and is hosted on Linaro infrastructure. This verification service can be used to verify CCA attestation tokens that come from emulated Arm platforms, including the example token that you have been using in this exercise.
+
+Linaro’s verification service is implemented using components from the open source Veraison project.
```

Comment:
Veraison in this sentence should link to https://github.com/veraison

Reply:
done